openpower/platform/Config.in

```diff
@@ -1,12 +1,14 @@
 config BR2_OPENPOWER_PLATFORM
-    bool "Enable the OpenPower Platform"
+    bool "Enables only one openPower platform"
     default n
-    depends on BR2_powerpc64 || BR2_powerpc64le
+    depends on ((BR2_OPENPOWER_POWER8 || BR2_OPENPOWER_POWER9) && !(BR2_OPENPOWER_POWER8 && BR2_OPENPOWER_POWER9))
```

Comment:
We need to have a way to check that both BR2_OPENPOWER_POWER8 and BR2_OPENPOWER_POWER9 are not set at the same time. I tried some other options (setting a string config) but this is the one that worked, albeit messy.

If we want to drop the whole check, we can do that as well since this is an issue that people shouldn't run into but we have to check for. And then revisit it at a later time